### ISSUE TITLE
Link to audio is wrong

### ISSUE THREAD

**Issue by Zz9uk3** [mysticcoders/voicenotes-sync#63]
When I tick the box to also download the audio of the transcript it is appearing properly however as soon as I click on the link which should lead me to the audio a new file is created instead:

![image](https://github.com/user-attachments/assets/ff49d4eb-f7db-4a98-948f-793837f23e03)

**Comment by Zz9uk3** [comment]
For the moment I worked around this bug by building the link to the mp3 manually instead of using {{audio_link}}:

```
{% if audio_link %}
[Audio](Informations/voicenotes/audio/{{recording_id}}.mp3)
{% endif %}
```

**Comment by kinabalu** [comment]
Will be releasing a beta with a fix for this shortly 0.0.30-beta2

**Comment by kinabalu** [comment]
this will be fixed when we release 0.0.30
